tests/query_runner/test_script.py

```diff
@@ -0,0 +1,11 @@
+from redash.query_runner.script import Script
+from tests import BaseTestCase
+
+
+class TestInit(BaseTestCase):
+    def test_wildcard(self):
+        runner = Script({'path': '*'})
+        self.assertIsInstance(runner, Script)
+
+    def test_outside(self):
+        self.assertRaises(ValueError, lambda: Script({'path': '../../'}))
```

Comment:
I know you just added a test, but if you want to take a stab at it: I think this functionality is needed when running a script and not when creating the query runner.

Reply:
@arikfr 

Thanks, but I don't have any good idea to run script for it.

I'm going to remove this file at this time, and I will send another PR when I have a good solution.